Homework6/task_1.py

```diff
@@ -0,0 +1,144 @@
+# Создайте  модель из жизни. Это может быть бронирование комнаты в отеле,
+# покупка билета в транспортной компании, или простая РПГ. Создайте
+# несколько объектов классов, которые описывают ситуацию Объекты должны
+# содержать как атрибуты так и методы класса для симуляции различных действий.
+# Программа должна инстанцировать объекты и эмулировать какую-либо
+# ситуацию - вызывать методы, взаимодействие объектов и т.д.
+
+
+class Hotel(object):
+    free_rooms = ['101', '102', '103', '104', '105']
+    rented_rooms = []
+    booked_rooms = []
+    list_of_guests = {}
+
+
+class Person(object):
+    def __init__(self, name):
+        self.name = name
+        self.rentroom = []
+        self.bookroom = []
+
+    def rent_room(self, *numbers_of_room):
+        for n in numbers_of_room:
+            if n not in Hotel.free_rooms:
```

Comment:
bad practise to put this rent to particular person. it seems hotel should have method rent with argument person